metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
Personally I am fine with both but don't prefer any calculations with a constant later. For e.g., 'self.MINUTE + self.MINUTE' and defining one here could lead to such a situation.